### PR TITLE
Rome Italy

### LINKED ISSUE #9: 8amlaser GitHub Workflow

### PR DESCRIPTION
**Pull request by 8amlaser**
I think this document can help people find stuff to do in Rome. :smile: :it: 

closes #9 

fixes #9 

resolve #9

### CODE REVIEW

File: Rome.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Rome, IT
```
Comment (hentschellj) [inline Rome.md:1]: @8amlaser One of my most desired places to visit.

**Comment by madepu81** [comment]
Wow Rome!!!... Do you recommend any place to stay there?

**Comment by madepu81** [comment]
But kind of hype... I heard differently from the people who went Rome :(  I hope i am not spoiling your enthusiasm.  So you are from SF?

**Comment by 8amlaser** [comment]
@madepu81 I was just born in Rome, so unfortunately I don't know of a good place where to stay...sorry! But I'd definitely recommend a visit...it's like walking around a living museum (and if this is good or bad, I'll leave it to you to decide!). I am currently visiting SF for half a year or so :smiley: You from Irvine?

**Pull request merged**